File: Backend/src/Acme.ChatAppss.Application.Contracts/Messages/MessageDto.cs
using System;
using System.Collections.Generic;
using System.Text;
using Volo.Abp.Application.Dtos;

namespace Acme.ChatApp.Messages
{
    public class MessageDto : AuditedEntityDto<Guid>
    {
        public Guid Id {  get; set; }
        public Guid SenderId { get; set; }
        public Guid? ReceiverId { get; set; }
        public Guid? GroupId { get; set; }
        public string Content { get; set; }
    }
}

File: Frontend/src/app/Dto/MessageDto.ts
export class MessageDto {
  id: any;
  senderId?: string;
  receiverId: string;
  content: string;
  groupId?: string;
  isGroup? : boolean;

  constructor() {
    this.senderId = '';
    this.receiverId = '';
    this.content = '';
    this.id = '';
    this.groupId = '';
    this.isGroup = false
  }
}
